Jointify/Model/PoseMachineLearning/PoseNet.swift

```diff
@@ -59,11 +59,16 @@ class PoseNet {
         static let jointColor: UIColor = UIColor.systemPink
         // Degrees that will be subtraced from the measured angle
         static let neutralNullAngle: Float = 90.0
+        // Minimum confidence that each joint has to conform to
+        static let confidenceThreshold: Double = 0.70
     }
     
     let side: Side
     private let jointSegments: [JointSegment]
+    private let selectedJointNames: [JointName]
     var degree: Float = 0.0
+    // Dictionary for the coordinates of all joints
+    var jointPositions = [String: Float]()
```

Comment:
Style guide proposes "When initializing an Array, always use the type annotation". 
```suggestion
    var jointPositions: [String: Float] = [:]
```